vm/src/obj/objbyteinner.rs

```diff
@@ -287,6 +287,10 @@ impl PyByteInner {
         self.cmp(other, |a, b| a == b, vm)
     }
 
+    pub fn ne(&self, other: PyObjectRef, vm: &VirtualMachine) -> PyComparisonValue {
+        self.cmp(other, |a, b| a != b, vm)
```

Comment:
when `__eq__` is implemented with `PyComparisionValue` return type , `__ne__` is idiomatically implemented like:

```suggestion
        self.eq(other, vm).map(|v| !v)
```

This implementation is applicable regardless of `__eq__` implementation.